gen/enum.go

```diff
@@ -101,6 +106,96 @@ func enum(g Generator, spec *compile.EnumSpec) error {
 			<end>
 			return fmt.Sprintf("<$enumName>(%d)", <$w>)
 		}
+
+		func (<$v> <$enumName>) MarshalJSON() ([]byte, error) {
+			<if len .Spec.Items>
+				switch int32(<$v>) {
+				<range .UniqueItems>
+					case <.Value>:
+						return ([]byte)("\"<.Name>\""), nil
```

Comment:
nit: no need for parens around `[]byte`, and you might want to use backtick quotes here,
```go
return []byte(`"<.Name>"`)
```

Also, is there any concern about the `.Name` containing quotes?